src/AWS/Orleans.Streaming.SQS/Storage/SQSStorage.cs

```diff
@@ -41,19 +51,26 @@ internal class SQSStorage
         /// </summary>
         /// <param name="loggerFactory">logger factory to use</param>
         /// <param name="queueName">The name of the queue</param>
-        /// <param name="connectionString">The connection string</param>
+        /// <param name="sqsOptions">The options for the SQS connection</param>
         /// <param name="serviceId">The service ID</param>
-        public SQSStorage(ILoggerFactory loggerFactory, string queueName, string connectionString, string serviceId = "")
+        public SQSStorage(ILoggerFactory loggerFactory, string queueName, SqsOptions sqsOptions, string serviceId = "")
```

Comment:
(Note for later, not to do in this PR)

We should inject the SQS Client directly here